README.md

```diff
@@ -22,6 +22,10 @@ It also compiles for Android but never been tested on it.
 
 The minimum-supported version fo `rustc` is **1.45**.
 
+### Use in binaries distributed on the MacOS App Store
+Apple has restrictions as to which APIs can be used in binaries distributed through the app store.
+By default, `sysinfo` is not compatiable with these restrictions. You can use the `apple-app-store` feature
+flag to disable the Apple prohibited features.
```

Comment:
Please add an empty line after this one.

Reply:
Empty line has been added.